style.css

```diff
@@ -1,36 +1,860 @@
+
+  .featured-speakers2 .grid {
+    display: grid;
+    grid-template-columns: 2fr 4fr;
+    gap: 5px;
+  }
+
+  .container3 {
+    max-width: 1100px;
+    margin: 0 auto;
+    padding: 0 20px;
+  }
+
+  hr.line2 {
+    border: 1px solid orangered;
+    width: 5%;
+    margin: auto;
+    margin-top: 20px;
+  }
+
+  .featured-speakers2 ul {
+    text-decoration: none;
+    list-style: none;
+  }
+
+  .featured-speakers2 ul h2 {
+    font-size: 24px;
+    font-weight: 700;
+    color: #272a31;
+  }
+
+  .featured-speakers2 .p1 {
+    font-size: 10px;
+    font-weight: 700;
+    color: orangered;
+    text-align: left;
+    padding-top: 0;
+  }
+
+  .featured-speakers2 .p2 {
+    font-size: 10px;
+    font-weight: 700;
+    color: #272a31;
+    text-align: left;
+    padding-top: 10px;
+  }
+
+  .featured-speakers ul h2 {
+    font-size: 15px;
+    font-weight: 700;
+    color: #272a31;
+  }
+
+  .featured-speakers .p1 {
+    font-size: 10px;
+    font-weight: 600;
+    color: orangered;
+    text-align: left;
+    padding-top: 10px;
+  }
+
+  .featured-speakers .p2 {
+    font-size: 10px;
+    font-weight: 700;
+    color: black;
+    text-align: left;
+    padding-top: 10px;
+  }
+
+  .more {
+    display: none;
+  }
+
+  .program-box:hover {
+    transform: scale(0.9);
+    border: solid orangered;
+  }
+
+  .Lines {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-right: 50%;
+    width: 100%;
+  }
+
+  .prog {
+    margin-left: 18%;
+    margin-top: 5%;
+    padding-bottom: 20px;
+  }
+
+  .f-par {
+    display: flex;
+    column-gap: 10%;
+  }
+
+  .f-par2 {
+    display: flex;
+    column-gap: 10%;
+  }
+
+  .par-flex {
+    display: flex;
+    column-gap: 10%;
+    padding-top: 15px;
+    margin-left: 18%;
+  }
+
+  .main-abt {
+    margin-left: 30%;
+  }
+
+  .main-abt .middle {
+    margin-top: 3%;
+    width: 70%;
+  }
+
+  #part {
+    display: none;
+  }
+
+  .partners {
+    background-color: #313236;
+    width: 100%;
+    color: #858585;
+    margin-top: 5%;
+  }
+
+  .head-p {
+    padding-top: 15%;
+  }
+
+  .footer {
+    display: flex;
+    justify-content: space-between;
+    font-family: "Lato", sans-serif;
+  }
+
+  .footer-flex {
+    width: 30%;
+  }
+
+  .footer-flex img {
+    width: 70%;
+  }
+
+  .about-second {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 8%;
+  }
+
+  .about-second img {
+    width: 50%;
+    border-width: 1px;
+  }
+
+  .about-par {
+    margin-top: 4%;
+  }
+
+  .about-child {
+    display: flex;
+    flex-direction: column;
+    margin-top: 5%;
+  }
+
+  .past-flex {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 5%;
+  }
+
+  .past-events {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    column-gap: 7%;
+    width: 50%;
+    margin-top: 3%;
+  }
```

Comment:
- [ ] Please we should try as much as possible to stick to the original design. Therefore I suggest that you apply the orange design and the texts on top of the image to match the original design. This will also improve the user Interface of your application ☺️.

Your design 😎 
![image](https://github.com/Zel-hub7/Capstone-project1/assets/99394374/4a9dbcca-9368-4f73-b1c9-165c1e5edb25)

Original design 😎 
![image](https://github.com/Zel-hub7/Capstone-project1/assets/99394374/aaa5852d-71ef-4cf3-9919-fe8081d1a4d4)